app/views/cases/data_request_areas/show.html.slim

```diff
@@ -44,7 +44,7 @@ div class="case"
             td
               span = @commissioning_document&.download_link
 
-      - if @data_request_area.data_request_emails.blank? && policy(@case).can_record_data_request?
+      - if @data_request_area.data_request_emails.blank? && policy(@case).can_record_data_request? && !@case.offender_sar_complaint?
```

Comment:
Maybe we can replace `policy(@case).can_record_data_request?` with a policy method that relates to sending the email and that will differ depending in whether the case is a complaint or not.

Reply:
There is an action of 'send_day_1_email' that is not enabled for complaints which is what causes the error, so that is what should be used to determine if sending is allowed.